apps/ex_shop/test/models/line_item_test.exs

```diff
@@ -82,10 +100,27 @@ defmodule ExShop.LineItemTest do
     product.master
   end
 
+  defp create_product_with_variant do
+    master_variant = create_product
+    product = master_variant |> Repo.preload([:product]) |> Map.get(:product)
```

Comment:
Can we please avoid by refactoring create_product (I skipped it :( ) to provide created product directly ??

or Add a TODO for now, need to revisit